tests/functional/spec/features/validation/date_validation/date-combined-yyyy.spec.js

```diff
@@ -8,7 +8,7 @@ describe("Feature: Combined question level and single validation for MM-YYYY dat
 
   describe("Period Validation", () => {
     describe("Given I enter dates", () => {
-      it("When I enter a single dates that are too early/late, Then I should see a single validation errors", () => {
+      it("When I enter a single dates that are too early/late, Then I should see single validation errors", () => {
```

Comment:
Think this description needs to be updated to something like:
```suggestion
      it("When I enter dates that are too early and too late, Then I should see two validation errors", () => {
```